int64 joint_number
float64[6] position
